modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSharedLogStorage.java

```diff
@@ -532,11 +548,13 @@ private void addConfBatch(LogEntry entry, WriteBatch batch) throws RocksDBExcept
      */
     private boolean executeBatch(WriteBatchTemplate template) {
         this.readLock.lock();
-        if (this.db == null) {
+
+        if (stopped) {
```

Comment:
I would prefer moving this part under the try-finally block

Reply:
Done